client/src/components/beer-item.jsx

```diff
@@ -8,7 +8,9 @@ export const BeerItem = ({ beer: { brewery, name, rating } }) => (
       <dt>Name</dt>
       <dd>{name}</dd>
       <dt>Rating</dt>
-      <dd>{rating}</dd>
+      <dd>{rating}</dd>  
```

Comment:
Looks like you've added some trailing space here. 
Might be good to look into a way of picking this up. Git will warn you about trailing space, but you can also install plugins in VS Code to highlight and/or trim trailing whitespace.
Actually prettier should fix this up so maybe you just need to run prettier over this file.

Reply:
Removed trailing space. Thanks for the reminder about formatting.